libs/libfstrm/Makefile

```diff
@@ -0,0 +1,64 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libfstrm
+PKG_VERSION:=0.5.0
+PKG_RELEASE=1
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/fstrm-$(PKG_VERSION)
+PKG_SOURCE:=fstrm-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://dl.farsightsecurity.com/dist/fstrm/
+PKG_HASH:=10ee7792a86face1d2271dc591652ab8c7af6976883887c69fdb11f10da135fc
+
+PKG_MAINTAINER:=James Taylor <james@jtaylor.id.au>
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
+
+PKG_INSTALL:=1
+
+PKG_FIXUP:=autoreconf
+
+include $(INCLUDE_DIR)/package.mk
+
+PKG_BUILD_PARALLEL:=1
+
+define Package/libfstrm
+  SECTION:=libs
+  CATEGORY:=Libraries
+  TITLE:=This is fstrm, a C implementation of the Frame Streams data transport protocol.
+endef
+
+define Package/libfstrm/description
+	fstrm is an optimized C implementation of Frame Streams that includes a fast,
+	lockless circular queue implementation and exposes library interfaces for
+	setting up a dedicated Frame Streams I/O thread and asynchronously submitting
+	data frames for transport from worker threads. It was originally written to
+	facilitate the addition of high speed binary logging to DNS servers written
+	in C using the dnstap log format.
+endef
+
+define Build/Configure
```

Comment:
I prefer using CONFIGURE_ARGS += instead of this.

Reply:
Yes, CONFIGURE_ARGS should be used

Reply:
Thanks, I've changed it to use CONFIGURE_ARGS +=